glib/src/object.rs

```diff
@@ -1264,15 +1264,19 @@ pub trait ObjectExt: ObjectType {
     where
         N: Into<&'a str>,
         F: Fn(&[Value]) -> Option<Value>;
-    fn emit<'a, N: Into<&'a str>>(
+    /// Emit signal by signal id.
+    fn emit(&self, signal_id: SignalId, args: &[&dyn ToValue]) -> Result<Option<Value>, BoolError>;
```

Comment:
Gir seems to generate code using emit("signal-name"...). Do it make sense to do the change in gir's side? :)

Reply:
Where would gir get the signal id from?

Reply:
change it as in replace the call with emit_by_name

Reply:
Ah, yes that makes sense :)